docs/dev/release.md

```diff
@@ -5,12 +5,9 @@ That page also explains concepts like *pre-releases* and applications thereof.
 
 ## Preparing the release
 
-1. Make a new branch off of `main` to prepare the release notes, and create a PR from this branch back into `main`.
-   Add a milestone for the desired version to be released.
-2. Update the date in the desired release’s notes and if applicable, delete empty headers in the notes.
-   Create a new blank note for the next desired release and update the `index.md` to include it.
-3. Push the changes to the PR, and merge into `main`.
-   If it is a patch release, backport the updated notes (see [](#versioning-tooling)) into the major/minor version branch.
+1. Switch to the `main` branch for a major/minor release and the respective release series branch for a *patch* release (e.g. `1.8.x` when releasing version 1.8.4).
+2. Run `hatch towncrier:build` to generate a PR that creates a new release notes file. Wait for the PR to be auto-merged.
+3. If it is a *patch* release, merge the backport PR (see {ref}`versioning-tooling`) into the `main` branch.
```

Comment:
Out of curiosity, why aren't backport PRs auto-merge by default?

Reply:
I guess because we haven’t configured the meeseeksbot to do that (if it can do that)

Reply:
https://github.com/scientific-python/MeeseeksDev/issues/132

Reply:
very sad...but thanks for opening the issue! would be great to have this!